### PR TITLE
yaml file 

### PR DESCRIPTION
**Pull request by spicy-rice**
- created in /pipelines/

### CODE REVIEW

File: front-end-components/pipelines/front-end.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+trigger:
+- main
+- feature/*
```
Comment (michael-fielding) [inline front-end-components/pipelines/front-end.yaml:3]: Don't think we want to trigger whenever you push to your feature branch. It should be when a change is made to main only

File: front-end-components/pipelines/front-end.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  displayName: 'Run tests'
+  inputs:
+    command: 'custom'
+    customCommand: 'test'
+
+- task: Npm@1
+  displayName: 'Build project'
+  inputs:
+    command: 'custom'
+    customCommand: 'run build'
```
Comment (michael-fielding) [inline front-end-components/pipelines/front-end.yaml:31]: Where is the step to deploy to the artifact pipeline?

File: front-end-components/pipelines/front-end.yaml
Comment (jrabbott) [inline front-end-components/pipelines/front-end.yaml:1]: trigger section needs updating to the following;
image

pr section needs updating to the following;
image

pool vm image should be 'windows-2022'

Add the following to each of the NPM steps;

`workingDir: 'front-end-components'`

Checkout step isn't required, therefore can be removed.

**Review by jrabbott — CHANGES_REQUESTED** [review]

**Review by jrabbott — CHANGES_REQUESTED** [review]
Need to review spacing / indentation. Has to be correct or the pipeline will fail.

**Review by jrabbott — APPROVED** [review]

File: front-end-components/pipelines/front-end.yaml (hunk 1 above)
Comment (jrabbott) [reply front-end-components/pipelines/front-end.yaml]: @michael-fielding I think this is resolved now?

**Pull request merged**